clang-tools-extra/clang-tidy/utils/Matchers.h

```diff
@@ -136,6 +128,15 @@ class MatchesAnyListedNameMatcher
     }
   };
 
+  bool matches(
+      const NamedDecl &Node, ast_matchers::internal::ASTMatchFinder *Finder,
+      ast_matchers::internal::BoundNodesTreeBuilder *Builder) const override {
+    return llvm::any_of(NameMatchers, [&Node](const NameMatcher &NM) {
+      return NM.match(Node);
+    });
+  }
+
+private:
```

Comment:
Why was this change needed? The other changes in the code do not indicate this function getting called anywhere.

Reply:
The diff here highlights the wrong part of the change:
https://github.com/llvm/llvm-project/pull/106350/commits/47c1a5694fed5769bb6bbc437e31276efe41d520

I made the contained NameMatcher class public, but the diff considered the other function to be moved.